bindings/python/tests/test_pt.py

```diff
@@ -1,33 +1,48 @@
+from pathlib import Path
 import unittest
 
 import torch
 
 from safetensors.safetensors_rust import safe_open
-from safetensors.torch import load_file, save_file
+from safetensors.torch import load, load_file, save, save_file
 
+this_dir = Path(__file__).parent
+data_dir = this_dir / "data"
 
 class TorchTestCase(unittest.TestCase):
-    def test_odd_dtype(self):
-        data = {
+    PT_MMAP_FILE = str(data_dir / "out_safe_pt_mmap_small.safetensors")  # FixMe: must accept Path!
```

Comment:
Do you mind not changing the style of this code ?

We don't need `pathlib` so it's not used on purpose (accepting it as an argument for users that prefer it, is on the other hand desirable).
I personally dislike pathlib because I have encountered bad bugs with it, and I feel it provides little value overall.

Overall your code hurts the very essence of what makes good tests according to me:

**Tests must be readable with no further context and the reader must able by purely reading this code if it's correct or not.**
If this property is not respected, and a reader cannot infer if the code is valid or not simply, then how is it better than the code it is supposed to test. If one has to think hard about correctness, then let's validate directly the actual functional code and skip the test.
That being said this is a principle, and sometimes it's not applicable. This is not the case here imho.

- This implies that repetition in tests is more than welcome, because as a reader I don't have to read 2, 10 or 100 functions to understand what the test is doing, all the context is there right in the current test I am reading.
- Here you introduce multiple levels of indirection, `cls` and `self._class` which I have no clue as a reader in which state they are in when reading locally. Did anyone modify them during the tests ? I have no clue.

I am happy to modify this code for you if you don't want to do it.